docs/explanation/security.rst

```diff
@@ -0,0 +1,49 @@
+:relatedlinks: [Ubuntu&#32;Security&#32;reporting&#32;and&#32;disclosure&#32;policy](https://ubuntu.com/security/disclosure-policy)
```

Comment:
The related links only appear in the right panel. It is also worth adding a section at the end, e.g. "Security reporting and disclosure", for reporting issues in this GitHub repo or through the Ubuntu org